.gitignore

```diff
@@ -1 +1,2 @@
 *-junit.xml
+tests/.idea/
```

Comment:
```suggestion
/tests/.idea/
```

Reply:
this change would not be appropriate... it is a relative path we are looking to ignore

Reply:
Actually, feel free to ignore this. Looks like for multi-component paths the semantics is the same.

I thought it differs like for simple paths, where `/foo` only ignores `foo` that is in the root of the repo, while `foo` ignores all `foo` files everywhere in the repo.